vllm/model_executor/layers/quantization/utils/marlin_utils.py

```diff
@@ -22,76 +22,69 @@
 USE_FP32_REDUCE_DEFAULT = True
 
 
-def _check_marlin_supported(num_bits: int, group_size: int, is_sym: bool,
-                            min_capability: Optional[int],
-                            has_zp: bool) -> Tuple[bool, Optional[str]]:
-    if min_capability is not None:
+# For binary size and compile time, we don't support the same types for with and
+#  without runtime zero-point. We support common cases, i.e. AWQ and GPTQ.
+#  TODO: we may want to move this into the C++ so its closer to the actual impl
+def marlin_supported_quant_types(has_zp: bool,
+                                 min_capability: Optional[int] = None):
```

Comment:
this is like a super nit but maybe better to name this like `query_marlin_supported_quant_types`, mainly to contrast with the fact that `GPTQ_MARLIN_24_SUPPORTED_QUANT_TYPES` is just a list and make it more obvious why one of these similar names is all caps and the other not.

Reply:
ya I like this, leaves room for maybe migrating to query the compiled c++ code in the future.

Done.